### ISSUE TITLE
specify arg and flag values via boa

### ISSUE THREAD

**Issue by bbkane** [elewis787/boa#9]
It's really neat that Boa allows for interactive subcommand selection, but (from the GIF), it looks like it exits after you pick the command. It would be really neat if you could use Boa to execute the selected subcommand with the args/flags you'd like. 

I'm writing my own Cobra alternative, so I'm super interested in watching your innovation here - I'll probably port it to mine :D

**Comment by elewis787** [comment]
I agree that this would be a great add - I've thought about fair amount and, for now, landed on just adding the print option. However, I actually think the better solve would be to use a text input that defaults for the command usage line and allows the user to add additional flags/options. 

I'll see what I can do !! Thanks for checking this out.

**Comment by reitzig** [comment]
FWIW, [Textualize/trogon](https://github.com/Textualize/trogon) may be a good source of inspiration.
